### PR TITLE
Feature/fro 87 remove auth

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Removing the auth wrapper and associated logic
- Opening up pages to not required a logged-in user
- Removal of all auth pages from the build
- Set up a redirect on any `/auth/...` pages to homepage

### CODE REVIEW

File: frontend/src/api/auth.ts [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
 
-const unprotectedUrls = [
-  '/auth/sign-in',
-  '/auth/sign-up',
-  '/auth/activate-account',
-  '/auth/reset-password',
-  '/auth/reset-request',
-];
+const unprotectedUrls = ["/auth/sign-in", "/auth/sign-up", "/auth/activate-account", "/auth/reset-password", "/auth/reset-request"];
+const protectedUrls = ["/account"];
```
Comment (joel-wright) [inline frontend/src/api/auth.ts]: I thought this was all going away?

File: frontend/src/api/http-common.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ class ApiClient {
       const token = storage.getToken();
       if (token) {
         // eslint-disable-next-line no-param-reassign
-        config.headers.common.Authorization = `Bearer ${token}`;
+        config.headers.common["Authorization"] = `Bearer ${token}`;
       }
       return config;
     });
```
Comment (joel-wright) [inline frontend/src/api/http-common.ts:22]: is this doing anything any more?

File: frontend/src/hooks/useOutsideAlerter.ts [hunk 3]
```diff
@@ -10,7 +10,6 @@ export default function useOutsideAlerter(ref, cb = (event) => {}) {
      */
     function handleClickOutside(event) {
       if (ref.current && !ref.current.contains(event.target)) {
-        // alert('You clicked outside of me!');
```
Comment (joel-wright) [inline frontend/src/hooks/useOutsideAlerter.ts:13]: :joy:

File: frontend/src/api/http-common.ts (hunk 2 above)
Comment (joel-wright) [inline frontend/src/api/http-common.ts:22]: Is this doing anything any more?

File: frontend/src/pages/privacy.tsx [hunk 4]
```diff
@@ -1,235 +1,143 @@
...
+          <p>
+            We also collect information through cookies and similar technologies. See our cookie policy <a href="/cookie-policy">here</a>.
+          </p>
+          <h2>How we use your information</h2>
+          <p>
+            Climate Policy Radar collects and processes your personal information, provided as detailed above, as necessary in order to provide you with our service and any related
+            services you may request, as well as to:
+          </p>
+          <ul>
+            <li>Facilitate account creation and login process</li>
```
Comment (joel-wright) [inline frontend/src/pages/privacy.tsx:59]: this may need to change

**Review by joel-wright — APPROVED** [review]
Looks good, just a couple of comments

File: frontend/src/api/auth.ts (hunk 1 above)
Comment (joel-wright) [reply frontend/src/api/auth.ts]: already gone!

File: frontend/src/api/http-common.ts (hunk 2 above)
Comment (joel-wright) [reply frontend/src/api/http-common.ts:22]: duplicated

File: frontend/src/pages/privacy.tsx (hunk 4 above)
Comment (PatFawbertMills) [reply frontend/src/pages/privacy.tsx:59]: Offline discussion to decide it is OK to remain for now as it covers the historic functionality.

**Review by joel-wright — APPROVED** [review]
Cleaning up all the things!! :sparkles:

**Pull request merged**